tests/runner/Cargo.toml

```diff
@@ -15,8 +15,11 @@ common-error.workspace = true
 common-query.workspace = true
 common-recordbatch.workspace = true
 common-time.workspace = true
+datatypes = { workspace = true }
+mysql = { version = "23.0.1" }
 serde.workspace = true
 serde_json.workspace = true
+tokio-postgres = { workspace = true }
```

Comment:
We are using `sqlx` in the integration test, is it possible to use it here as well?

Reply:
With sqlx text we have to handle many decode logic to convert value into text. I'm not sure whether it's as convenient as this two crates. Since `sqlness` somehow independent to the core part,  I think it's ok to introduce a new dependency?

Reply:
> We are using `sqlx` in the integration test, is it possible to use it here as well?

We can merge this PR first and create another issue to discuss it. @waynexia @J0HN50N133